pos_order_mgmt/static/src/js/widgets.js

```diff
@@ -210,12 +210,24 @@ odoo.define('pos_order_mgmt.widgets', function (require) {
             order.trigger('change');
             this.pos.get('orders').add(order);
             this.pos.set('selectedOrder', order);
+            order.get_orderlines().forEach(function (orderline) {
+                if (orderline.pack_lot_lines)
+                {
+                    _.each(orderline.return_pack_lot_names, function(lot_name) {
+                        orderline.pack_lot_lines.add(new models.Packlotline(
+                            {'lot_name': lot_name}, {'order_line': orderline}
+                        ));
+                    })
+                    order.save_to_db();
+                    orderline.trigger('change', orderline);
+                }
+            });
             return order;
         },
 
         _prepare_order_from_order_data: function (order_data, action) {
             var self = this;
-            var order = new pos.Order({}, {
+            var order = new models.Order({}, {
```

Comment:
Not a big deal, but this kind of pointless changes misslead the target of the PR, don't you think :)

Reply:
Well.. I needed to reference `models.Packlotline` and this way it's much more readable than `pos.Packlotline`. Specially because `pos` is used everywhere.
IMO, it should've always been `models`, so I don't consider it pointless

Reply:
In favor to change. pos is not the good name, and is error prone.